multiqc/modules/peddy/peddy.py

```diff
@@ -205,3 +212,31 @@ def peddy_relatedness_plot(self):
                 <span style="color: #2B9F2B;">greather than 0.5</span>.""",
                 plot = scatter.plot(data, pconfig)
             )
+            
+    def peddy_het_check_plot(self):
+        """plot the het_check scatter plot"""
+        # empty dictionary to add sample names, and dictionary of values
+        data = {}
+        # for each sample, and list in self.peddy_data
+        for s_name, d in self.peddy_data.items():
+            # check the sample contains the required columns
+            if 'median_depth_het_check' in d and 'het_ratio_het_check' in d:
+                # add sample to dictionary with value as a dictionary of points to plot
+                data[s_name] = {
+                    'x': d['median_depth_het_check'],
+                    'y': d['het_ratio_het_check']
+                }
+        
+        pconfig = {
+            'id': 'peddy_het_check_plot',
+            'title': 'Peddy Het Check Plot',
+            'xlab': 'median depth',
+            'ylab': 'proportion het calls',
+            }
+    
+        self.add_section (
+            name = 'Peddy Het Check Plot',
```

Comment:
This section is already under Peddy in the nav, so no need to repeat this again - `Het Check` should be fine.

Reply:
Also, please add at least a `description` and preferably also a longer `helptext` for this section.